habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -543,26 +563,26 @@ def place_robot_at_closest_point_with_navmesh(
 
 
 def _get_robot_spawns(
-    target_position: np.ndarray,
     rotation_perturbation_noise: float,
     distance_threshold: float,
     sim,
     num_spawn_attempts: int,
     filter_colliding_states: bool,
+    target_position: Optional[np.ndarray] = None,
```

Comment:
Based on my comment above about using existing function to randomly place the agent, this shouldn't be changed.

Reply:
Done, revert back to the original design